website/docs/r/function_node.html.markdown

```diff
@@ -43,7 +43,7 @@ resource "aci_function_node" "example" {
 - `is_copy` - (Optional) If the device is a copy device.
   Allowed values: "yes", "no". Default value: "no".
 - `managed` - (Optional) Specified if the function is using a managed device.
-  Allowed values: "yes", "no". Default value: "yes".
+  Allowed values: "yes", "no". Default value: "no".
```

Comment:
Check on version prior to 5.2 and on 5.2 to confirm behavior.

Reply:
ok